app/models/reviewable_queued_post.rb

```diff
@@ -35,20 +41,9 @@ def updatable_reviewable_scores
   def build_actions(actions, guardian, args)
     unless approved?
       if topic&.closed?
-        actions.add(:approve_post_closed) do |a|
-          a.icon = "check"
-          a.label = "reviewables.actions.approve_post.title"
-          a.confirm_message = "reviewables.actions.approve_post.confirm_closed"
-          a.completed_message = "reviewables.actions.approve_post.complete"
-        end
+        build_action(actions, :approve_post_closed, icon: "check", confirm: true)
```

Comment:
The `reviewables.actions.approve_post_closed.description` i18n key description is not set for this action. I'm wondering how the existing code handles this and whether it will result in any missing translation messages being shown on the client side.

Reply:
No missing translation message is shown, it's handled by `ReviewableActionSerializer`.